odbc_diag_record.c

```diff
@@ -511,12 +511,37 @@ odbc_get_diag_field (SQLSMALLINT handle_type,
 	case SQL_DIAG_CONNECTION_NAME:	/* yet not implemeted */
 	  if (diag_info_ptr && buffer_length > 0)
 	  {
+	    int connhd = -1;
+		int cci_ret = 0;

```

Comment:
CUBRID Coding Convention is not kept, 
1. make proper indentation
2. use brace for both if-then-else statement even it is single statement.
    if ( cond )
    {
    }
    else statement

   if (cond)
     {
     }
   else
     {
     }